tests/test_evaluator.py

```diff
@@ -207,49 +257,76 @@ def test_model_init(self):
             label_column=self.label_column,
             label_mapping=self.label_mapping,
         )
-        self.assertEqual(scores, {"accuracy": 0})
+        self.assertEqual(results["accuracy"], 0)
 
     def test_class_init(self):
         evaluator = ImageClassificationEvaluator()
         self.assertEqual(evaluator.task, "image-classification")
         self.assertIsNone(evaluator.default_metric_name)
 
-        scores = evaluator.compute(
+        results = evaluator.compute(
             model_or_pipeline=self.pipe,
             data=self.data,
             metric="accuracy",
             input_column=self.input_column,
             label_column=self.label_column,
             label_mapping=self.label_mapping,
         )
-        self.assertEqual(scores, {"accuracy": 0})
+        self.assertEqual(results["accuracy"], 0)
 
     def test_default_pipe_init(self):
-        scores = self.evaluator.compute(
+        results = self.evaluator.compute(
             data=self.data,
             input_column=self.input_column,
             label_column=self.label_column,
             label_mapping=self.label_mapping,
         )
-        self.assertEqual(scores, {"accuracy": 0})
+        self.assertEqual(results["accuracy"], 0)
 
     def test_overwrite_default_metric(self):
         accuracy = load("accuracy")
-        scores = self.evaluator.compute(
+        results = self.evaluator.compute(
             model_or_pipeline=self.pipe,
             data=self.data,
             metric=accuracy,
             input_column=self.input_column,
             label_column=self.label_column,
             label_mapping=self.label_mapping,
         )
-        self.assertEqual(scores, {"accuracy": 0})
-        scores = self.evaluator.compute(
+        self.assertEqual(results["accuracy"], 0)
+        results = self.evaluator.compute(
             model_or_pipeline=self.pipe,
             data=self.data,
             metric="accuracy",
             input_column=self.input_column,
             label_column=self.label_column,
             label_mapping=self.label_mapping,
         )
-        self.assertEqual(scores, {"accuracy": 0})
+        self.assertEqual(results["accuracy"], 0)
+
+
+class TestEvaluatorPerf(TestCase):
```

Comment:
This seems a bit redundant with the `test_perf`/`test_bootstrap_and_perf`? I think we can just keep those tests as part of the the main test suite and don't need an extra test case for now. Wdyt? Then we don't need to define everything again.